### ISSUE TITLE
[FEATURE] New XML identifiers

### ISSUE THREAD

**Issue by EngThis** [Cingulara/openrmf-docs#201]
This Feature request is in relation to new XML identifiers that will be used in STIGS/SRGs, the move by DISA to the new identifiers is not yet complete.

**Describe the solution you'd like**
Review and identify the changes that will be coming so that OpenRMF can continue to be of use.

**Describe alternatives you've considered**
Does not seem like there are alternatives...

**Additional context**
An excerpt from the Email sent by DISA
"As noted in a recent news announcement, to provide increased flexibility for the future, DISA is updating the systems that produce Security Technical Implementation Guides (STIGs) and Security Requirements Guides (SRGs). The initial modification will be to change Group and Rule IDs (Vul and Subvul IDs). The previous Group and Rule IDs will be retained through the update as “legacy” IDs, presented as XCCDF ident elements. See the example below:
<Group id="V-204392">
  <title>SRG-OS-000257-GPOS-00098</title>
  <description>…</description>
  <Rule id="SV-204392r85825_rule" weight="10.0" severity="high">
    <version>RHEL-07-010010</version>
    <title>The Red Hat Enterprise Linux operating system must be configured so that the file permissions, ownership, and group membership of system files and commands match the vendor values.</title>
    <description>…</description>
    <reference>…</reference>
    <ident system="http://cyber.mil/legacy">SV-86473</ident>
    <ident system="http://cyber.mil/legacy">V-71849</ident>
    <ident system="http://cyber.mil/cci">CCI-001494</ident>
    <ident system="http://cyber.mil/cci">CCI-001496</ident>
    <ident system="http://cyber.mil/cci">CCI-002165</ident>
    <ident system="http://cyber.mil/cci">CCI-002235</ident>"

The email states that the following test STIGS are available for testing/review on https://public.cyber.mil/stigs/downloads/

•	Microsoft Windows Server 2019 TEST STIG - Ver 2, Rel 0.3 
•	Microsoft Windows Server 2019 TEST STIG Benchmark - Ver 2, Rel 0.3 
•	Red Hat Enterprise Linux 7 TEST STIG - Ver 3, Rel 0.3 
•	Red Hat Enterprise Linux 7 TEST STIG Benchmark - Ver 3, Rel 0.3 

Also if you have not already done so I would suggest subscribing to receive STIG update emails from DISA at https://public.govdelivery.com/ sign up and then Look for STIG Updates under assorted topics.

You may have already had this information, I did not see a feature request in regards to this and wanted to make sure its not overlooked.

Thanks for the hard work.

**Comment by Cingulara** [comment]
Yes, we do subscribe to the updates. We have this on our radar but we did not put into Issues yet. I am waiting on the final to come out. I have gone through the initial REL and Windows STIG xccdf manual checklist file and the structure we parse now seems to match that structure. I have walked through that in code and the XML.

The upgrade process however will need to be matched. When there is a new Release or Version out, the system warns you and says "hey you can upgrade this" but with the newer system may not match the VULN IDs correctly I believe. That will have to be fixed and tested heavily.

**Comment by Cingulara** [comment]
This is in the latest release as of December 13, 2020
